app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -112,6 +112,9 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
         case Task.statusFailed:
           log.debug('Using ${task.status} from commit ${task.commitSha} ref ${task.ref} builder ${task.builderName}');
           return GithubBuildStatusUpdate.statusFailure;
+        case Task.statusInfraFailure:
+          log.debug('Using ${task.status} from commit ${task.commitSha} ref ${task.ref} builder ${task.builderName}');
+          return GithubBuildStatusUpdate.statusFailure;
```

Comment:
nit: these can be combined to
```suggestion
        case Task.statusInfraFailure:
          log.debug('Using ${task.status} from commit ${task.commitSha} ref ${task.ref} builder ${task.builderName}');
          return GithubBuildStatusUpdate.statusFailure;
```

Reply:
Thanks! Updated.